collections/_article/dev-snapshot-godot-4-5-dev-2.md

```diff
@@ -0,0 +1,114 @@
+- Editor: Improve default/no query quick open dialog behavior ([GH-104061](https://github.com/godotengine/godot/pull/104061)).
+- Editor: Remove New prefix from EditorResourcePicker ([GH-104604](https://github.com/godotengine/godot/pull/104604)).
+- Export: Updates and fixes to the Android prebuilt export logic ([GH-103173](https://github.com/godotengine/godot/pull/103173)).
+- GDScript: Return early when parsing invalid super call ([GH-104509](https://github.com/godotengine/godot/pull/104509)).
+- GUI: Improve Popup `content_scale_factor` ([GH-104399](https://github.com/godotengine/godot/pull/104399)).
+- GUI: Optimize startup times by using `ubrk_clone` instead of `ubrk_open` ([GH-104455](https://github.com/godotengine/godot/pull/104455)).
+- GUI: Scroll `EditorInspector` while drag & drop hovering near the edges ([GH-103943](https://github.com/godotengine/godot/pull/103943)).
+- Import: Load decompressable texture format if no supported one is found ([GH-104590](https://github.com/godotengine/godot/pull/104590)).
+- Navigation: Allow compiling templates without navigation features ([GH-104811](https://github.com/godotengine/godot/pull/104811)).
+- Physics: Allow compiling templates without physics servers ([GH-103373](https://github.com/godotengine/godot/pull/103373)).
+- Physics: Jolt: Update to 5.3.0 ([GH-104449](https://github.com/godotengine/godot/pull/104449)).
+- Porting: Android: Add an editor setting to enable/disable `TouchActionsPanel` ([GH-105015](https://github.com/godotengine/godot/pull/105015)).
+- Porting: Android: Add support for `Mute Game` toggle([GH-104409](https://github.com/godotengine/godot/pull/104409)).
+- Porting: Linux: Detect KDE/LXQt and swap OK/Cancel buttons to Windows style ([GH-104959](https://github.com/godotengine/godot/pull/104959)).
+- Porting: macOS: Replace custom main loop with `[NSApp run]` and `CFRunLoop` observer ([GH-104397](https://github.com/godotengine/godot/pull/104397)).
+- Porting: macOS: Support more controllers on macOS 11+ ([GH-104619](https://github.com/godotengine/godot/pull/104619)).
+- Rendering: Avoid using a global variable to store instance index in canvas items shader in RD renderer ([GH-105037](https://github.com/godotengine/godot/pull/105037)).
+- Rendering: Implement Fragment density map support ([GH-99551](https://github.com/godotengine/godot/pull/99551)).
+- XR: Deactivate the `CameraServer` by default ([GH-104232](https://github.com/godotengine/godot/pull/104232)).
+- XR: OpenXR: Clean-up `OpenXRExtensionWrapper` by removing multiple inheritance and deprecating `OpenXRExtensionWrapperExtension` ([GH-104087](https://github.com/godotengine/godot/pull/104087)).
+
+## Changelog
+
+**90 contributors** submitted **250 fixes** for this release. See our [**interactive changelog**](https://godotengine.github.io/godot-interactive-changelog/#4.5-dev2) for the complete list of changes since the 4.4-stable.
```

Comment:
```suggestion
**90 contributors** submitted **250 fixes** for this release. See our [**interactive changelog**](https://godotengine.github.io/godot-interactive-changelog/#4.5-dev2) for the complete list of changes since the previous 4.5-dev1 snapshot.
```